RELEASE.md

```diff
@@ -1,3 +1,65 @@
+# Release 1.5.0
+
+## Breaking Changes
+* Prebuilt binaries are now built against CUDA 9 and cuDNN 7.
+* Our Linux binaries are built using ubuntu 16 containers, potentially
+  introducing glibc incompatibility issues with ubuntu 14.
+* Starting from 1.6 release, our prebuilt binaries will use AVX instructions.
+  This may break TF on older CPUs.
+
+## Major Features And Improvements
+* Eager execution preview version is available under `tensorflow/contrib/eager/`.
```

Comment:
Should we make this a link?
[Eager execution](https://research.googleblog.com/2017/10/eager-execution-imperative-define-by.html)

And the same for TensorFlow lite, linking to https://www.tensorflow.org/mobile/tflite/?